osu.Game.Rulesets.Mania.Tests/Mods/TestSceneManiaModFadeIn.cs

```diff
@@ -15,5 +15,8 @@ public partial class TestSceneManiaModFadeIn : ModTestScene
         [TestCase(0.1f)]
         [TestCase(0.7f)]
         public void TestCoverage(float coverage) => CreateModTest(new ModTestData { Mod = new ManiaModFadeIn { Coverage = { Value = coverage } }, PassCondition = () => true });
+
+        [Test]
+        public void TestComboBasedCoverage([Values] bool coverage) => CreateModTest(new ModTestData { Mod = new ManiaModFadeIn { ComboBasedCoverage = { Value = coverage } }, PassCondition = () => true });
```

Comment:
Rename param to `enabled` or `comboBased` or something. Same goes for `TestSceneManiaModHidden`.